packages/graphql-hooks/src/useClientRequest.js

```diff
@@ -51,6 +51,13 @@ function reducer(state, action) {
   opts.skipCache: Boolean
 */
 function useClientRequest(query, initialOpts = {}) {
+  if (typeof query !== 'string') {
+    /* eslint-disable-next-line no-console */
+    console.warn(
```

Comment:
@good-idea Since this will fail, we should throw an error.

Your warning message is great, let's use it for the error message - perhaps add at the end add `remove it from your query/mutation.` just in case the developer is confused if the error comes from `useMutation`.